pathetic-nms/paper/src/main/java/org/patheloper/nms/NMSInterface_Paper.java

```diff
@@ -0,0 +1,20 @@
+package org.patheloper.nms;
+
+import org.bukkit.Bukkit;
+import org.bukkit.ChunkSnapshot;
+import org.bukkit.World;
+import org.patheloper.api.snapshot.NMSInterface;
+
+public class NMSInterface_Paper implements NMSInterface {
```

Comment:
If we are trying to stick to proper naming conventions there shouldn't be Underscores in class names